src/storage/FirebaseStorageDirective.js

```diff
@@ -0,0 +1,29 @@
+/* istanbul ignore next */
+(function () {
+  "use strict";
+
+  function FirebaseStorageDirective($firebaseStorage, firebase) {
+    return {
+      restrict: 'A',
+      priority: 99, // run after the attributes are interpolated
+      scope: {},
+      link: function (scope, element, attrs) {
+        // $observe is like $watch but it waits for interpolation
+        // Ex: <img firebase-src="{{ myUrl }}"/>
+        attrs.$observe('firebaseSrc', function (newFirebaseSrcVal) {
+          if (newFirebaseSrcVal !== '' && newFirebaseSrcVal !== null && newFirebaseSrcVal !== undefined) {
+            var storageRef = firebase.storage().ref(newFirebaseSrcVal);
```

Comment:
If we are passing `newFirebaseSrcVal` to `ref()`, it must be a string. So, should the above conditional be `typeof newFirebaseSrcVal === 'string' && newFirebaseSrcVal !== ''`?